retinanet/train.py

```diff
@@ -16,28 +16,27 @@
 def train(model, state, path, annotations, val_path, val_annotations, resize, max_size, jitter, batch_size, iterations, val_iterations, mixed_precision, lr, warmup, milestones, gamma, is_master=True, world=1, use_dali=True, verbose=True, metrics_url=None, logdir=None):
     'Train the model on the given dataset'
 
-    # Prepare dataset
-    if verbose: print('Preparing dataset...')
-    data_iterator = (DaliDataIterator if use_dali else DataIterator)(
-        path, jitter, max_size, batch_size, model.stride,
-        world, annotations, training=True)
-    if verbose: print(data_iterator)
-
     # Prepare model
     nn_model = model
```

Comment:
You can remove this if we're not using it anymore

Reply:
nn_model is still used here for saving the model checkpoint (without the DDP wrapper): https://github.com/NVIDIA/retinanet-examples/pull/20/files/780ca6e9f989ce88b3bdaca88f7784066362eb12#diff-9339c352aabfb452a77c9714dac89004R146

Reply:
Also, just did a test for resuming training and it appears to work fine.